### DISCUSSION TITLE
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED]

### DISCUSSION THREAD

**Discussion by PhoebeLZ** [explosion/spaCy#11624] [Help: Installation]
When I try to type python -m spacy download en_core_web_sm in the command prompt, I received an error message:

```
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1056)
During handling of the above exception, another exception occurred:
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /explosion/spacy-models/master/compatibility.json (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1056)')))
During handling of the above exception, another exception occurred:
requests.exceptions.SSLError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /explosion/spacy-models/master/compatibility.json (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1056)')))
```

I am wondering what is the problem here. 
Thank you.

**Comment by polm** [comment]
What version of spaCy/Python are you using? How did you install spaCy? Are you by any chance behind a corporate proxy?

We have seen errors like this before, but they are usually due to either network issues or old versions of libraries used for downloading the packages. 

It's usually easy to fix by upgrading the appropriate libraries, but without knowing what the exact cause is, it's probably easiest to download the model from the [model page](https://spacy.io/models/en) (click on "Release Details").
